ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -17,6 +18,13 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
   @service session;
   @service store;
 
+  // =attributes
+
+  @tracked targets = this.model;
```

Comment:
Do you think it's possible that we have a code-smell here by having to save the model and not just using the model directly returned from the route? Don't we have a convention to do the data fetching in routes?

How will we handle loading or error cases with this?

Reply:
Great suggestion. Updated!